save-orchestrator/src/main/kotlin/com/saveourtool/save/orchestrator/service/DockerService.kt

```diff
@@ -243,20 +243,17 @@ class DockerService(private val configProperties: ConfigProperties,
             log.info("Base image [$baseImage] for execution ${execution.id} doesn't exists, will build it first")
             buildBaseImage(sdk)
         }
-        val aptCmd = "apt-get ${configProperties.aptExtraFlags}"
         val imageId = dockerContainerManager.buildImageWithResources(
             baseImage = baseImage,
             imageName = imageName(execution.id!!),
             baseDir = resourcesPath,
             resourcesTargetPath = executionDir,
-            runCmd = """RUN $aptCmd update && env DEBIAN_FRONTEND="noninteractive" $aptCmd install -y \
```

Comment:
Could you please describe the reason of changing the order of calling these `RUN` commands? now they will be called only at the end of function

Reply:
I've moved them into base image build only, because these commands are required for every execution. I.e. from logs of `DockerServiceTest`, first build is 
```
13:47:41.345 [Test worker] DEBUG c.s.s.o.d.DockerContainerManager - Using generated Dockerfile FROM docker.io/library/ubuntu:latest
RUN apt-get  update && env DEBIAN_FRONTEND="noninteractive" apt-get  install -y \
libcurl4-openssl-dev tzdata
RUN ln -fs /usr/share/zoneinfo/UTC /etc/localtime
RUN rm -rf /var/lib/apt/lists/*

RUN useradd --create-home --shell /bin/sh save-agent
WORKDIR /home/save-agent/save-execution
```
and the second is 
```
13:48:01.049 [Test worker] DEBUG c.s.s.o.d.DockerContainerManager - Using generated Dockerfile FROM docker.io/library/save-base-ubuntu:latest

COPY resources /home/save-agent/save-execution
RUN chmod +x /home/save-agent/save-execution/save-agent.kexe
RUN chmod +x /home/save-agent/save-execution/save-0.3.2-linuxX64.kexe
RUN chown -R save-agent .
USER save-agent
```